internal/scaffold/ansible/watches.go

```diff
@@ -44,8 +44,8 @@ const watchesAnsibleTmpl = `---
   group: [[.Resource.FullGroup]]
   kind: [[.Resource.Kind]]
   [[- if .GeneratePlaybook ]]
-  playbook: /opt/ansible/playbook.yml
+  playbook: playbook.yml
   [[- else ]]
-  role: /opt/ansible/[[.RolesDir]]/[[.Resource.LowerKind]]
+  role: [[.RolesDir]]/[[.Resource.LowerKind]]
```

Comment:
I'd just specify `[[.Resource.LowerKind]]` and let ansible handle discovering the default RolesDir based on the `ANSIBLE_ROLES_PATH`